doc/gui_x11.jax

```diff
@@ -52,6 +52,8 @@ GUIが開始されるときには、fork()して現在のプロセスを終了
 は、":gui -b" を使用するとよい。
 
 "gvim --nofork" と "gvim -f" は同じである。
+
+実行中のジョブがないときは、子プロセスではないので Vim は fork しない。
```

Comment:
「ジョブがないとき」
あるときじゃないでしょうか？
よく分からないですが、こんな感じ？
「プロセスが子プロセスではなくなるため、実行中のジョブがある場合、Vim は fork しない。」

Reply:
ping @k-takata

Reply:
@h-east さん、@k-takata さん、
いつもレビューありがとうございます。
最近ちょっとドタバタしていて修正遅くなってしまうかもしれません。。
こちらの指摘事項に対しては理解いたしました！
たぶん、翻訳時の単なる読み間違いだと思います。

Reply:
> When there are running jobs Vim will not fork, because the processes would no
longer be child processes.

私ならこう訳すかな？単に原文と語順を合わせただけですが。
「実行中のジョブがある場合、Vim は fork しない。なぜならそれらのプロセスが子プロセスではなくなってしまうためである。」